.github/dependabot.yml

```
@@ -0,0 +1,6 @@
+version: 2
+updates:
+  - package-ecosystem: "pip"
+    directory: "/"
+    schedule:
+      interval: "daily"
```

```suggestion
version: 2
updates:
  - package-ecosystem: "pip"
    directory: "/"
    schedule:
      interval: "daily"
    labels:
      - "maintenance"
    open-pull-requests-limit: 5
```

This will do as stated – I think reducing the number of open pull requests would help reduce spam. In any case, I would be happy to help review these PRs, if I were given triage rights or similar.